src/widgets/dock_area/show/leaf.rs

```diff
@@ -797,7 +802,7 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
         }
 
         //change hover destination
-        if let Some(pointer) = ui.input(|i| i.pointer.hover_pos()) {
+        if let Some(pointer) = state.hover_pos {
```

Comment:
It might be good to replace all other instances of `input.pointer.hover_pos()` with `state.last_hover_pos`, to make this the new established pattern for others to follow. Of course, verify that everything you change still works as intended.

Reply:
I did, however it is only really necessary, when something should happen on the frame, where the touch is released. On all other frames, the new `state.last_hover_pos` will be the same as `ui.input(|i| i.pointer.hover_pos())`